usb_cdc:
  ros__parameters:
    vendor_id: 0x0483
    product_id: 0x5740
    publish_period_ms: 20
    send_period_ms: 20
